app/js/validateUserAdditions.js

```diff
@@ -48,16 +48,17 @@ async function getExistingUsers() {
  * validates email using regex code
  *
  * @param email - the email address we want to check for
+ * @param event - the event that's fired off
```

Comment:
What is this? It is deffo not a parameter

Reply:
resolved - I think